documentation/src/docs/controller-service.md

```diff
@@ -173,7 +173,7 @@ interface. This includes API calls for Stream management as well as
 other administration API primarily dealing with _creation_ and _deletion_ of
 [**Scopes**](pravega-concepts.md#streams). We use [swagger](https://swagger.io) to describe our `REST` API. Please see, the swagger [`yaml`](https://github.com/pravega/pravega/tree/master/shared/controller-api/src/main/swagger) file.
 
-## Pravega Controller Service
+## Pravega Controller Service <a name ="controller-service"></a>
```

Comment:
Why is there nothing inside of this link tag?

Reply:
@tkaitchuck @shiveshr As we had two sections having the title `Pravega controller service`....mismatch happened and the link was not rendered correctly so i included html tag for one of the section.... problem resolved.